models/repo.go

```diff
@@ -1219,6 +1244,19 @@ type CreateRepoOptions struct {
 	Status      RepositoryStatus
 }
 
+// GenerateRepoOptions contains the template units to generate
+type GenerateRepoOptions struct {
+	Name        string
+	Description string
+	Private     bool
+	GitContent  bool
```

Comment:
So what _is_ actually copied when `GitContent` is false?

Reply:
Theoretically issue labels, avatar, etc as they are added in later PRs.  
In this PR, it can never be false since it's the only option and at least one option must be selected.

I admit I don't like adding "in case it's implemented later" code, so I can definitely change this if needed.

Reply:
> 
> I admit I don't like adding "in case it's implemented later" code, so I can definitely change this if needed.

I'd just _hide_ it from the UI; it has no current purpose. But I'm totally with you about laying the path for these features that we already know we want.

Reply:
So rather than comment out or hide things for a future PR, I instead decided to add one more unit to justify it.  
Topics just happened to be low-hanging fruit.